### DISCUSSION TITLE
How get rid of NFtables rules from OpenSnitch with Firewall interaction disabled ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#893] [Q&A] [answered]
Hi everyone,

My system have few nftables rules That I would like to keep (and / or not invalided with other pushed rules )

So in OpenSnitch I tick off Firewall
![FW_OFF](https://user-images.githubusercontent.com/23012674/228894417-5ac84a14-fa9d-4022-a2a7-13be0cc60ed7.png)

Nevertheless I see that those rules are been pushed in my nftables ruleset

```shell
table inet filter {
	chain input {
		type filter hook input priority filter; policy accept;
		udp sport 53 queue num 0 bypass
	}
}
table inet nat {
}
table inet mangle {
	chain output {
		type filter hook output priority mangle; policy accept;
		ct state related,new queue num 0 bypass
	}
}
```
I really would like to get rid of:
```shell
table inet filter {
	chain input {
		type filter hook input priority filter; policy accept; # ! O_o
		udp sport 53 queue num 0 bypass
	}
}
table inet nat {
}
```

The last one:
```shell
table inet mangle {
	chain output {
		type filter hook output priority mangle; policy accept;
		ct state related,new queue num 0 bypass
	}
}
```
Will only let traffic out if an OpenSnitch rule allow it.. right ?

btw OpenSnitch version is : 160rc5

Thanks

**Comment by BobSquarePants** [comment]
Was already happening in 1.6.0rc2
https://github.com/evilsocket/opensnitch/issues/710

**Reply by gustavo-iniguez-goya** [reply, marked as answer]
Hi @BobSquarePants ,

Those are the interception rules. If you want to get rid of them, disable the interception, but OpenSnitch wouldn't intercept connections then.
